server-spi-private/src/main/java/org/keycloak/migration/migrators/MigrateTo16_0_0.java

```diff
@@ -0,0 +1,39 @@
+package org.keycloak.migration.migrators;
+
+import org.keycloak.migration.ModelVersion;
+import org.keycloak.models.AccountRoles;
+import org.keycloak.models.ClientModel;
+import org.keycloak.models.Constants;
+import org.keycloak.models.KeycloakSession;
+import org.keycloak.models.RealmModel;
+import org.keycloak.models.RoleModel;
+import org.keycloak.representations.idm.RealmRepresentation;
+
+public class MigrateTo16_0_0 implements Migration {
```

Comment:
Since we've completely forgot about this PR and time has moved on this will have to be updated to 20_0_0

Reply:
Done together with all related  migration tests.